components/Activity.tsx

```diff
@@ -4,7 +4,7 @@ import { useEffect, useState } from 'react';
 import InfiniteScroll from 'react-infinite-scroll-component';
 
 import { Card, NoDataContainer } from 'components/shared/styles';
-import { GNOSIS_SCAN_URL } from 'constants/index';
+import { GNOSIS_SCAN_URL, NA } from 'constants/index';
```

Comment:
Unrelated comment – not at all a blocker for this PR. It's great that we have NA as a constant, alas I don't think the way we're writing "NA" is conventional. Should really be "n/a"

Reply:
agree, updated